### DISCUSSION TITLE
Individual registeration for the GitHub Developer Program

### DISCUSSION THREAD

**Discussion by abdoulsn** [community/community#24333] [Apps, API and Webhooks]
<p>Hello I’d like to know if it’s possible to register for the GitHub Developer Program with my individual account?</p>

**Comment by canuckjacq** [comment]
<p>Yes - you’ll be able to choose the account you register with here:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/developer/register" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:500/500;"><img src="https://user-images.githubusercontent.com/11161530/181084267-af9a3f10-930a-4c18-b904-3c7449243d92.png" class="thumbnail" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png"></div>

<h3><a href="https://github.com/developer/register" target="_blank" rel="noopener">Build software better, together</a></h3>

<p><a href="https://github.com/developer/register" target="_blank" rel="noopener">oper/register</a></p>

  <p><span class="label1">GitHub is where people build software. More than 50 million people use GitHub to discover, fork, and contribute to over 100 million projects.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by abdoulsn** [comment]
<p>This ask company mail and website. My question is if I’m individual person who want to join the program is that possible?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Membership to the program is open to individual developers and companies who have:</p>
<ul>
<li>An integration in production or development using the GitHub API</li>
<li>An email address where GitHub users can contact you for support</li>
</ul>
<p>If you don’t have a website, you can always create a GitHub Pages site for your integration.</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://pages.github.com/favicon.ico" class="site-icon" width="16" height="16">
      <a href="https://pages.github.com/" target="_blank" rel="noopener">GitHub Pages</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://pages.github.com/" target="_blank" rel="noopener">GitHub Pages</a></h3>

<p>Websites for you and your projects, hosted directly from your GitHub repository. Just edit, push, and your changes are live.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by abdoulsn** [comment]
<p>Thanks I create I’ve a GitHub page.</p>
